src/layouts/AppContainer.tsx

```diff
@@ -97,35 +97,37 @@ class AppContainer extends React.Component<IProps, null> {
     return (
       (connectionStatus === ConnectionStatus.Pending
         ? <div className={css.loading}>Loading...</div>
-      : connectionStatus === ConnectionStatus.Failed
-        ? <NoWeb3Container />
-      : ethAccountAddress === null
-        ? <NoEthAccountContainer />
-      : connectionStatus == ConnectionStatus.Connected
-        ? <div className={css.outer}>
-            <div className={css.container}>
-              <Route path="/dao/:daoAddress" children={(props) => (
-                <HeaderContainer daoAddress={props.match ? props.match.params.daoAddress : null} />
-              )} />
-              <Switch>
-                <Route exact path="/" component={HomeContainer}/>
-                <Route exact path="/dao/create" component={CreateDaoContainer}/>
-                <Route path="/dao/:daoAddress" component={ViewDaoContainer}/>
-                <Route exact path="/proposal/create/:daoAddress" component={CreateProposalContainer}/>
-              </Switch>
-            </div>
-            <div className={css.notificationsWrapper}>
-              {Object.keys(operations).map((key) => ({key, operation: operations[key]})).map(({key, operation}) =>
-                (<Notification
-                  key={key}
-                  operation={operation}
-                  close={() => dismissOperation(key)}
-                />),
-              )}
-            </div>
-            <div className={css.background}></div>
-          </div>
-        : <div className={css.loading}>Something weird happened, please contact the DAOstack team...</div>
+        : connectionStatus === ConnectionStatus.Failed
+          ? <NoWeb3Container />
+          : ethAccountAddress === null
+            ? <NoEthAccountContainer />
```

Comment:
hmm, i find this indentation more confusing, why do you like it better?

Reply:
vscode auto format...